assets/src/dashboard/components/storyAnimation/provider.js

```diff
@@ -114,7 +115,16 @@ function Provider({ animations, children, onWAAPIFinish }) {
 
   const WAAPIAnimationMethods = useMemo(() => {
     const play = () =>
-      WAAPIAnimations.forEach((animation) => animation?.play());
+      WAAPIAnimations.forEach((animation) => {
+        // Sometimes an animation part can get into a
+        // stuck state where executing `play` doesn't
+        // trigger the animation. A workaround to avoid
+        // this is to first `cancel` the animation
+        // before playing.
+        animation?.cancel();
+
+        animation?.play();
```

Comment:
This doesn't seem to have an adverse affect on playing other animations (long or short) and spamming the play button seems to work like before... however, feel free to double check and push back if you foresee this causing legit issues in the future.

Reply:
This works for me!

I think the only issue I can see in the future is if someone wants to scrub and play from the point they've scrubbed to, but this might not even be part of the product spec, so works for me for now. 

Got a good comment there too, so if it does become a case we need to handle, we know were to look and can figure out some other duct tape solution since we can't really fix the tool.